olive/systems/azureml/aml_system.py

```diff
@@ -81,7 +83,10 @@ def __init__(
         self._assert_not_none(aml_docker_config)
         aml_docker_config = validate_config(aml_docker_config, AzureMLDockerConfig)
         azureml_client_config = validate_config(azureml_client_config, AzureMLClientConfig)
+        if vc_config:
+            vc_config = validate_config(vc_config, VCConfig)
         self.azureml_client_config = azureml_client_config
+        self.vc_config = vc_config
```

Comment:
simplify to?
`self.vc_config = validate_config(vc_config, VCConfig) if vc_config else None`

Reply:
Done.

Reply:
There is no need to validate vc_config. Just pass it as it is resource. Let user provide proper dict.

Reply:
@devang-ml this validation only turns a dict to VCConfig object. We only validate `instance_type` here.